docs/framework/react/quick-start.md

```diff
@@ -3,54 +3,132 @@ id: quick-start
 title: Quick Start
 ---
 
-The bare minimum to get started with TanStack Form is to create a form and add a field. Keep in mind that this example does not include any validation or error handling... yet.
+TanStack Form is unlike most form libraries you've used before. It's designed for large-scale production usage, with a focus on type safety, performance, and developer experience.
+
+As a result, we've developed [a philosophy around the library's usage](/form/latest/docs/philosophy) that values scalability and long-term developer experience over short and sharable code snippets.
+
+Here's an example of a form following many of our best practices, which will allow you to rapidly develop even high-complexity forms after a short onboarding experience:
 
 ```tsx
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import { useForm } from '@tanstack/react-form'
+import { createFormHook, createFormHookContexts } from '@tanstack/react-form'
+import { TextField, NumberField, SubmitButton } from '~our-app/ui-library'
```

Comment:
I'd like to see one of those as an example (can you do multiple tabs in code fields?) or at least a link/mention towards the composition page to explain what these would look like.

Reply:
No tabs for our docs site, unfortunately, but ya fair enough. I'll link out to the composition page